umbra-js/hardhat.config.ts

```diff
@@ -11,6 +11,7 @@ import '@nomiclabs/hardhat-waffle';
 const chainIds = {
   ganache: 1337,
   goerli: 5,
+  sepolia: 11155111,
```

Comment:
I know this PR is a WIP, just skimming through
```suggestion
  sepolia: 11155111,
```

Reply:
Left it in cause i saw rinkeby, ropsten and kovan are still there. Should I remove those as well?

Reply:
we only need one testnet and sepolia is the future so I think removing goerli at the same time in this PR is ok, @apbendi lmk if you disagree

Reply:
Do you mean wait to merge this PR until the backend supports sepolia (which I assumed was the plan) or keep both for some period of time? This PR would swap batch send support from goerli to sepolia, if we remove goerli

Reply:
What I'm saying is, let's just leave removing Goerli support as something separate from this PR, to be followed up on quickly after everything is in place for Sepolia, so we don't have to do too much coordination with the backend changes.